Components/Modal/modal.css

```diff
@@ -0,0 +1,47 @@
+
+.modal {
+    display: none;
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    transform: scale(1.1);
+  }
+  .show-modal {
+    display: block;
+  }
+  .dialog{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    max-width: 300px;
```

Comment:
Try to be consistent while using units. You should use rem unit. So that for it will be helpful while making responsive website.

Reply:
okay, will change it.